avocado/plugins/yaml_to_mux.py

```diff
@@ -400,15 +400,14 @@ def run(self, args):
             else:
                 args.mux_out = out
 
+        data = MuxTreeNodeDebug() if args.mux.debug else MuxTreeNode()
+
         # Merge the multiplex
         multiplex_files = getattr(args, "mux_yaml", None)
         if multiplex_files:
             debug = getattr(args, "mux_debug", False)
             try:
-                data = create_from_yaml(multiplex_files, debug)
-                data = apply_filters(data, getattr(args, 'mux_only', None),
-                                     getattr(args, 'mux_out', None))
-                args.mux.data_merge(data)
+                data.merge(create_from_yaml(multiplex_files, debug))
```

Comment:
If you talk about the param names, then it's probably older change... I'll fix it in the next version...